pkgs/games/gnushogi/default.nix

```diff
@@ -10,6 +10,13 @@ stdenv.mkDerivation rec {
     sha256 = "0a9bsl2nbnb138lq0h14jfc5xvz7hpb2bcsj4mjn6g1hcsl4ik0y";
   };
 
+  env.LDFLAGS = "-Wl,-z,muldefs";
```

Comment:
```suggestion
  env.LDFLAGS = lib.optionalString (!stdenv.isDarwin) "-Wl,-z,muldefs";
```

Reply:
Does that fix darwin? If so please remove the mark broken again.

Reply:
No, this doesn't fix darwin, I couldn't find an alternative for `-z muldefs` for clang from searching.